statement.md

```diff
@@ -0,0 +1,84 @@
+# JupyterLab Accessibility Statement
+
+## Edited from the [W3C accessibility statement generator](https://www.w3.org/WAI/planning/statements/generator/#create)
+
+
+## ⚠️ Draft Accessibility Statement for JupyterLab
```

Comment:
```suggestion
## :construction: Draft Accessibility Statement for JupyterLab
```
I have realised that the other emoji renders widely different across OS and editors and is a nightmare

Reply:
I also know the `:emoji:` format doesn't render everywhere, but we can do this if it's preferred.